010-quarkus-opentracing-reactive-grpc/src/test/java/io/quarkus/qe/AbstractPingPongResourceTest.java

```diff
@@ -0,0 +1,70 @@
+package io.quarkus.qe;
+
+import static io.restassured.RestAssured.given;
+import static org.awaitility.Awaitility.await;
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.Matchers.allOf;
+import static org.hamcrest.Matchers.containsString;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.util.concurrent.TimeUnit;
+
+import org.apache.http.HttpStatus;
+import org.eclipse.microprofile.config.inject.ConfigProperty;
+import org.junit.jupiter.api.Test;
+
+import io.quarkus.qe.containers.JaegerTestResource;
+import io.quarkus.test.common.QuarkusTestResource;
+
+@QuarkusTestResource(JaegerTestResource.class)
+public abstract class AbstractPingPongResourceTest {
+
+    private static final String PING_ENDPOINT = "/%s-ping";
+    private static final String PONG_ENDPOINT = "/%s-pong";
+
+    private static final String PING_RESOURCE = "PingResource";
+    private static final String PONG_RESOURCE = "PongResource";
+
+    @ConfigProperty(name = JaegerTestResource.JAEGER_API_ENDPOINT, defaultValue = "http://localhost:16686/api/traces")
+    String jaegerEndpoint;
```

Comment:
This property is injected once the Jaeger container is up and running, but I had to provide a default value.